distutils/unixccompiler.py

```diff
@@ -139,6 +139,15 @@ class UnixCCompiler(CCompiler):
     if sys.platform == "cygwin":
         exe_extension = ".exe"
 
+    def _fix_lib_args(self, libraries, library_dirs, runtime_library_dirs):
+        """Remove standard library path from rpath"""
+        libraries, library_dirs, runtime_library_dirs = super()._fix_lib_args(
+            libraries, library_dirs, runtime_library_dirs)
+        libdir = sysconfig.get_config_var('LIBDIR')
+        if runtime_library_dirs and (libdir in runtime_library_dirs):
+            runtime_library_dirs.remove(libdir)
```

Comment:
```suggestion
        with contextlib.suppress(ValueError):
            runtime_library_dirs.remove(sys.config.get_config_var('LIBDIR')
```

Reply:
This leaves the case where `LIBDIR` isn't in `LD_LIBRARY_PATH`, which I suspect is relevant for `conda` envs on Linux; would this still produce a working extension in that case?

Reply:
```suggestion
        with contextlib.suppress(ValueError):
            runtime_library_dirs.remove(sysconfig.get_config_var('LIBDIR'))
```
and then also `import contextlib` at top might work better.

Reply:
> This leaves the case where [`LIBDIR` isn't in the default `LD_LIBRARY_PATH` or equivalent](https://github.com/pypa/distutils/pull/160#issuecomment-1200324907), which I suspect is relevant for `conda` envs on Linux; would this still produce a working extension in that case?

I'm confused by this comment. Is this comment pertaining to my suggested change, or an independent comment about the same section of code?

Reply:
I misunderstood the purpose of the suggestion at first; that is an independent comment